docs/faq.md

```diff
@@ -16,21 +16,19 @@ The Dusty daemon, which runs as root, manages your GitHub repos by default.
 To get permissions to clone from GitHub, the
 Daemon will make use of your unprivileged user's SSH_AUTH_SOCK. This means that the daemon's
 behavior using git should be the same as your user's, in terms of SSH authentication.
+(See [this GitHub help article](https://help.github.com/articles/generating-ssh-keys/)
+for some info about setting up GitHub ssh keys.)
```

Comment:
I think we shouldn't actually send people here. Running an additional ssh-agent like it suggests fucks with things, plus it doesn't tell you to use `-K`.